src/shims/manual/mscorlib.csproj

```diff
@@ -9,8 +9,9 @@
   <ItemGroup>
     <Compile Include="mscorlib.forwards.cs" />
     <ReferencePath
-      Include="$(RuntimePath)System.*.dll;$(RuntimePath)Microsoft.*.dll"
-      Exclude="$(RuntimePath)Microsoft.Diagnostics.Tracing.TraceEvent.dll;$(RuntimePath)Microsoft.DiaSymReader.Native.*.dll" />
+      Include="$(_RuntimePath)System.*.dll;$(_RuntimePath)Microsoft.*.dll"
+      Exclude="$(_RuntimePath)Microsoft.Diagnostics.Tracing.TraceEvent.dll;$(_RuntimePath)Microsoft.DiaSymReader.Native.*.dll;$(_RuntimePath)System.ValueTuple.dll" />
```

Comment:
@ericstj excluding System.ValueTuple from here will hurt the 2.2/2.1 facade generation? Should we exclude it for 2.0 only?

Reply:
Yes, you can build the excuse list as an item and condition the valuetuple exclusion